### PR TITLE
ignore camel case keywords in rule `value-keyword-case`

### PR DESCRIPTION
**Pull request by evilebottnawi**
https://github.com/stylelint/stylelint/issues/1089

### CODE REVIEW

File: src/rules/value-keyword-case/index.js [hunk 1]
```diff
@@ -20,6 +20,12 @@ const ignoredCharacters = new Set([
   "+", "-", "/", "*", "%",
 ])
 
+const ignoredCamelCaseFunctionNames = {
```
Comment (jeddy3) [inline src/rules/value-keyword-case/index.js]: `ignoredCamelCaseKeywords`

File: src/rules/value-keyword-case/index.js [hunk 2]
```diff
@@ -20,6 +20,12 @@ const ignoredCharacters = new Set([
...
+  "optimizespeed": "optimizeSpeed",
+  "optimizelegibility": "optimizeLegibility",
+  "geometricprecision": "geometricPrecision",
```
Comment (jeddy3) [inline src/rules/value-keyword-case/index.js]: Can move `currentColor` from the conditional and into here please?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# HEAD
+
+- Fixed: `value-keyword-case` now accounts for camelCase function names (e.g. `optimizeSpeed`, `optimizeLegibility` and `geometricPrecision`) when the `lower` option is used.
```
Comment (jeddy3) [inline CHANGELOG.md]:
```md
- Fixed: `value-keyword-case` now accounts for camelCase keywords (e.g. `optimizeSpeed`, `optimizeLegibility` and `geometricPrecision`) when the `lower` option is used."
```

**Comment by evilebottnawi** [comment]
@jeddy3 ping

**Comment by jeddy3** [comment]
Looks great!

**Pull request merged**